SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -41,40 +41,42 @@ public enum TransactionResult {
 }
 
 public protocol PaymentQueue: class {
-
+    
     func add(_ observer: SKPaymentTransactionObserver)
     func remove(_ observer: SKPaymentTransactionObserver)
-
+    
     func add(_ payment: SKPayment)
     
     func start(_ downloads: [SKDownload])
     func pause(_ downloads: [SKDownload])
-    #if os(watchOS)
-    func resumeDownloads(_ downloads: [SKDownload])
```

Comment:
See below comments...

Reply:
Its better to just fix it by adding it to the extension so we only need one #if.  see line 60